extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1590,6 +1601,20 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperationWithProgress<QuickStartProjectResult, QuickStartProjectProgress> GenerateAsync();
     }
 
+    // Extensions implement this and is provided to Dev Home when CreateChatStyleGenerationOperation is called.
+    // This allows to have an async operation can be started but also additional context reported back to Dev Home
+    // as progress is being made such as a detailed log out output through the adaptive card session.
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
+    interface IQuickStartChatStyleGenerationOperation
+    {
+        // Starts the operation which generates a project based on a given prompt by the user in the
+        // given output folder. The generated content in the output folder can be anything that is needed
```

Comment:
Is this comment accurate or the wrong one?

Reply:
If the contract version is being incremented here, we may also need to change the VersionOfSDK value inside build/azure-pipelines.yml.

Reply:
This comment is from an old one, good catch, should be fixed now. 

Also, resolving this conversation but I'm moving Sarah's comment onto the other thread where we talk about contract version.